pkg/schedule/schedulers/hot_region.go

```diff
@@ -625,16 +626,21 @@ func (bs *balanceSolver) solve() []*operator.Operator {
 			return region.GetStorePeer(srcStoreID) == nil
 		}
 	}
-
+	snapshotFilter := filter.NewSnapshotSendFilter(bs.GetStores(), constant.Medium)
```

Comment:
BTW, please add more comments about the priority.

Reply:
add comment for the user.